src/Model/Command/UserRecommendation.php

```diff
@@ -78,12 +80,14 @@ public function enableHardRotation(): self
     /**
      * Define threshold of how much relevant must the recommended items be to be returned.
      * Default minimal relevance is "low".
+     *
+     * @return static
```

Comment:
Again `$this`? Also below

Reply:
`@return static` and `@return $this` mean the same thing 🤔

Reply:
`static` tells its just the same *type* (resolved by LSB).
`$this` tells its not only the same type, but also the same actual instance. This means $this is "subset" of static, but not the same thing, and cannot be interchanged.

Its also recognized like this in proposed phpdoc PSR standard, see https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc.md#keyword - points 14. and 15.

Reply:
Cool, didn't know that. Fixed.